rql.go

```diff
@@ -275,7 +279,15 @@ func (p *Parser) parseField(sf reflect.StructField) error {
 			p.Log("Ignoring unknown option %q in struct tag", opt)
 		}
 	}
-	var filterOps []Op
+
+	if len(f.Name) == 0 && p.FieldNameFn != nil {
```

Comment:
```suggestion
	if f.Name != "" && p.FieldNameFn != nil {
```
It's easier to read the code, and get the `Name`'s type when using `!= ""`.